model/project_parser.go

```diff
@@ -644,40 +697,64 @@ func LoadProjectInto(ctx context.Context, data []byte, opts *GetProjectOpts, ide
 		return nil, errors.Wrapf(err, LoadProjectError)
 	}
 
-	// return intermediateProject even if we run into issues to show merge progress
-	for _, path := range intermediateProject.Include {
+	if len(intermediateProject.Include) > 0 {
 		if opts == nil {
 			err = errors.New("trying to open include files with empty options")
 			return nil, errors.Wrapf(err, LoadProjectError)
 		}
-		opts.UpdateForFile(path.FileName)
-
-		var yaml []byte
-		opts.Identifier = identifier
-		opts.RemotePath = path.FileName
-		grip.Debug(message.Fields{
-			"message":     "retrieving included YAML file",
-			"remote_path": opts.RemotePath,
-			"read_from":   opts.ReadFileFrom,
-			"module":      path.Module,
-		})
-		if path.Module != "" {
-			yaml, err = retrieveFileForModule(ctx, *opts, intermediateProject.Modules, path.Module)
-		} else {
-			yaml, err = retrieveFile(ctx, *opts)
+		wg := sync.WaitGroup{}
+		yamlChan := make(chan yamlTuple, len(intermediateProject.Include))
+		includeChan := make(chan Include)
+
+		// Be polite. Don't make more than 10 concurrent requests to GitHub.
+		for i := 0; i < 10; i++ {
```

Comment:
nit: since this is a magic and arbitrarily-chosen number, can this be a `const`?

Reply:
Fixed, though I wasn't sure if you'd have an opinion about where the const is defined. I defined it inline.

Reply:
Defining it inline is fine, just naming a var representing our arbitrary limit is good enough.